src/oas3/__tests__/__snapshots__/operation.test.ts.snap

```diff
@@ -1,5 +1,64 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
+exports[`transformOas3Operation callback 1`] = `
+Object {
+  "callbacks": Array [
+    Object {
+      "callbackName": "myCallback",
+      "id": "?http-operation-id?",
+      "iid": "cbId",
+      "method": "post",
+      "path": "http://prism.com?transactionId={$request.body#/id}",
```

Comment:
Let's not use domains we dont own. if its not localhost or example.org we could have weird things happening accidentally talking to other peoples servers.

Reply:
`http-spec` has no logic for making any requests but you right, let's keep it a general rule and use only safe domains. Changed 👍